curiouslearning/FeedTheMonsterJS
# fic: Cr 187 bangla json rebuild

# Changes
- lang/bangla/ftm_bangla.json
- /build

# How to test
- we can test to make sure that levels 25, 26, and 27 no longer have the “ghost character” issue 

Ref: [CR-187](https://curiouslearning.atlassian.net/jira/software/projects/CR/boards/7?selectedIssue=CR-187)


[CR-187]: https://curiouslearning.atlassian.net/browse/CR-187?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
